android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java

```diff
@@ -351,7 +350,7 @@ protected String decryptBytes(@NonNull final Key key, @NonNull final byte[] byte
     throws GeneralSecurityException, IOException {
     final Cipher cipher = getCachedInstance();
 
-    // decrypt the bytes using a CipherInputStream
+    // decrypt the bytes using cipher.doFinal()
```

Comment:
Might be good to add a comment in the code with a link to the issue, or this PR, describing why we don't use input streams. Otherwise someone might be eager to refactor this again, without testing on a pixel device first :)

Reply:
No problem at, makes complete sense to me.